assets/js/state/sagas/clusters.js

```diff
@@ -0,0 +1,24 @@
+import { put, takeEvery } from 'redux-saga/effects';
+import { CLUSTER_DEREGISTERED, removeCluster } from '@state/clusters';
+import { appendEntryToLiveFeed } from '@state/liveFeed';
+import { notify } from '@state/actions/notifications';
+
+export function* clusterDeregistered({ payload: { name, cluster_id } }) {
+  yield put(removeCluster({ id: cluster_id }));
```

Comment:
It was not simply better to change the projection code to use `id` instead of `cluster_id`?
To avoid transformations that we might have issues finding in the future